package.json

```diff
@@ -16,7 +16,7 @@
     "html-minifier": "^3.5.5",
     "minimalcss": "^0.3.1",
     "mkdirp": "^0.5.1",
-    "puppeteer": "^0.11.0",
+    "puppeteer": "^0.13.0",
```

Comment:
this is blocked by https://github.com/peterbe/minimalcss/pull/42

Reply:
So should it remain 0.11.0 then?

Reply:
yes for now